src/views/SignMessage.vue

```diff
@@ -39,7 +39,17 @@ import { Utf8Tools } from '@nimiq/utils';
 import { ERROR_CANCELED } from '@/lib/Constants';
 import { State as RpcState } from '@nimiq/rpc';
 
-@Component({components: {SmallPage, AccountSelector}})
+@Component({
+    components: {SmallPage, AccountSelector},
+    filters: {
+        withoutContracts(wallets: WalletInfo[]): WalletInfo[] {
+            return wallets.map((wallet) => {
+                wallet.contracts = [];
```

Comment:
I think this has side effects that might even change content in the store.
Please use a computed property that returns a filtered copy or introduce a flag `withoutContracts` on the `AccountSelector`.

Reply:
Yep, I fear Daniel's right. I was too quick approving this, sorry. My first thought was actually a flag in AccountsSelector. But at whatever place, Object.assign to the rescue :)

Reply:
Yes, I was aware that the filter affects the source object, but since contracts are never/nowhere used in during the SignMessage account selection and the Vuex store does have no effect on the IndexedDB, I thought this to be a non-problem.